nri/index.html

```diff
@@ -1 +1,207 @@
 <!DOCTYPE html>
+<html lang="ja">
+    <head>
+        <meta charset="utf-8">
+        <title>NRI</title>
+        <meta name="discription" content="NRIの公式ホームページです">
+
+    <!--CSS-->
+        <link rel="stylesheet" href="https://unpkg.com/ress/dist/ress.min.css">
+        <link href="https://fonts.googleapis.com/css?family=Philosopher" rel="stylesheet">
+        <link rel="icon" type="image/png" href="images/NRI.png">
+    </head>
+
+    <body>
+          <header class="page-header">
+              <h1><a href="#"><img class="logo" src="images/NRI.png" alt="NRIロゴ"></a></h1>
+              <nav>
+                <ul class="main-nav">
+                    <li><a href="#">ナレッジ・インサイト</a></li>
+                    <li><a href="#">ソリューション・サービス</a></li>
+                    <li><a href="#">サステナビリティ</a></li>
+                    <li><a href="#">ニュース</a></li>
+                    <li><a href="#">企業情報</a></li>
+                    <li><a href="#">株主・投資家情報（IR）</a></li>
+                    <li><a href="#">採用情報</a></li>
+                </ul>
+                <img src="images/search.png" alt="検索">
+                <img src="images/maru.png" alt="言語">
+                <img src="images/san.png" alt="メニュー">
+              </nav>
+          </header>
+          <ul class="log">
+            <li><a href="#">NRI トップ</a></li>
+            <li><a href="#">企業情報</a></li>
+            <li>NRIの強み：NRIのデジタルビジネス戦略</li>
+          </ul>
+          <div class="wrapper">
+            <h3 class="white-1">NRIの強み</h3>
+            <h1 class="white-2">NRIのデジタルビジネス戦略</h1>
+          </div><!--/.wrapper-->
+          <article>
+            <header class="post-info">
+              <h2 class="post-title">NRIのデジタルビジネス戦略</h2>
+            </header>
+            <p>
+              NRI では、企業の DX に向けた取り組みについて、既存ビジネスの業務プロセス変革に寄与する DX を
+              「DX1.0」、デジタルで新しいビジネスモデルそのものを生み出す DX を「DX2.0」と定義しています。
```

Comment:
めちゃくちゃ細かいのですが、「を」の後に改行がはいっていることで本番とテキストが一致していません。
修正をお願いします。

（本番のテキストをコピーして、こちらのコードに貼り付けたらヒットしなかった、という調べ方で発見しています。）

```suggestion
              NRI では、企業の DX に向けた取り組みについて、既存ビジネスの業務プロセス変革に寄与する DX を           「DX1.0」、デジタルで新しいビジネスモデルそのものを生み出す DX を「DX2.0」と定義しています。
```